react/global.css

```diff
@@ -133,6 +133,16 @@
   margin: 0 auto;
 }
 
+.vtex-gallery__row {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
```

Comment:
cant use rem?

Reply:
Done, I didn't think could be used here, thanks